### ISSUE TITLE
Push.register error - library not initialized

### ISSUE THREAD

**Issue by bradmartin** [Kinvey/nativescript-sdk#38]
#### Steps to Reproduce
call `Push.register()`will end up throwing the error from the core client:

```
KinveyError: You have not initialized the library. Please call Kinvey.init() to initialize the library.
```

even though in the app.component it has been initialized with the `init` method with 
```
   Kinvey.init({
      appKey: `${UserService.Kinvey_App_Key}`,
      appSecret: `${UserService.Kinvey_App_Secret}`
    });
```
which is successful as we are utilizing many parts of Kinvey in the app and right after a `ping()` will also resolve correctly.

Once I navigate to a page in the app (after the `init` and app start) this will throw when trying to `register`.

#### Expected Result

Method to resolve, not throw.

#### Sample Code that demonstrates the issue

```typescript
KinveyPush.register({
          android: {
            senderID: '01201921XXXX'
          },
          ios: {
            alert: true,
            badge: true,
            sound: true
          }
        })
          .then((deviceToken: string) => {
            console.log(`registered push notifications using KinveyPush with token: ${deviceToken}`);
            KinveyPush.onNotification((data: any) => {
              alert(`Message received!\n${JSON.stringify(data)}`);
            });
          })
          .catch((error: Error) => {
            console.log(`Couldn't register push notifications: ${error}`);
          });
```
#### Version of Kinvey NativeScript SDK

3.11.4

#### Kinvey AppKey

`kid_SyIIDJjdM`

#### NodeJS version
`8.9.1`

**Comment by bradmartin** [comment]
Also note, I've tried wrapping in a timeout to ensure it's not a 'timing' issue with no luck.

**Comment by thomasconner** [comment]
@bradmartin This has been fixed with [3.11.5](
https://github.com/Kinvey/nativescript-sdk/releases/tag/v3.11.5) release.
